browser/images/lc_toggledarktheme.svg

```diff
@@ -1,11 +1,11 @@
- </g>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketchjs="https://sketch.io/dtd/" version="1.1" sketchjs:metadata="eyJuYW1lIjoiNWI2M2NkNmYtYzE4Ni00YzI3LWExYjEtZWNjMWUyY2EyNGI2LnNrZXRjaHBhZCIsInN1cmZhY2UiOnsiaXNQYWludCI6dHJ1ZSwibWV0aG9kIjoiZmlsbCIsImJsZW5kIjoibm9ybWFsIiwiZW5hYmxlZCI6dHJ1ZSwib3BhY2l0eSI6MSwidHlwZSI6InBhdHRlcm4iLCJwYXR0ZXJuIjp7InR5cGUiOiJwYXR0ZXJuIiwicmVmbGVjdCI6Im5vLXJlZmxlY3QiLCJyZXBlYXQiOiJyZXBlYXQiLCJzbW9vdGhpbmciOmZhbHNlLCJzcmMiOiJ0cmFuc3BhcmVudExpZ2h0Iiwic3giOjEsInN5IjoxLCJ4MCI6MC41LCJ4MSI6MSwieTAiOjAuNSwieTEiOjF9LCJpc0ZpbGwiOnRydWV9LCJjbGlwUGF0aCI6eyJlbmFibGVkIjp0cnVlLCJzdHlsZSI6eyJzdHJva2VTdHlsZSI6ImJsYWNrIiwibGluZVdpZHRoIjoxfX0sImRlc2NyaXB0aW9uIjoiTWFkZSB3aXRoIFNrZXRjaHBhZCIsIm1ldGFkYXRhIjp7fSwiZXhwb3J0RFBJIjo3MiwiZXhwb3J0Rm9ybWF0IjoicG5nIiwiZXhwb3J0UXVhbGl0eSI6MC45NSwidW5pdHMiOiJweCIsIndpZHRoIjoyMDQ4LCJoZWlnaHQiOjk2OSwicGFnZXMiOlt7IndpZHRoIjoyMDQ4LCJoZWlnaHQiOjk2OX1dLCJ1dWlkIjoiNDdlYzczOTEtYmI5Ny00NTg2LWE1NzAtM2YxOGIxYmU1Y2JiIn0=" viewBox="909.8 315.83 314.83 314.83">

```

Comment:
The dimensions seem quite big and not corresponding to W: 24px and H: 24px. Could you please use 24px as icon size? Also it seems there is a lot of unneeded metadata from sketchjs. It would be good to either export it as plain SVG or alternatively edited it by hand so, at the end we have something like:

```svg
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path...
```

Reply:
I've resized the image and shortened the SVG.